plugins.json

```
@@ -575,7 +575,7 @@
     "name": "hyper-shellect",
     "description": "A short and shweet shell shellection menu.",
     "type": "plugin",
-    "preview": "https://raw.githubusercontent.com/joskore/hyper-shellect/master/shellect-preview.png",
+    "preview": "https://raw.githubusercontent.com/joskore/hyper-shellect/master/hyper-shellect.gif",
```

If you don't mind, I'd like to link the image via the commit it was added. If the image is removed or renamed in `master` it will break the image here, otherwise 😅 
```suggestion
    "preview": "https://raw.githubusercontent.com/joskore/hyper-shellect/ddcc8a379fb13d9a5af60f9f952e184c77328bcc/hyper-shellect.gif",
```